From cbda3a139de13a2b2752112ba73e5462e056a51c Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@gmail.com>
Date: Mon, 14 Oct 2019 21:55:42 +0300
Subject: [PATCH 5/9] check_boot_jars: Add IFAA Manager to whitelist

Fixes:
Error: out/target/common/obj/JAVA_LIBRARIES/org.ifaa.android.manager_intermediates/classes.jar
  contains class file org/ifaa/android/manager/IFAAManager.class,
  whose package name org.ifaa.android.manager is not in the whitelist
  build/make/core/tasks/check_boot_jars/package_whitelist.txt
  of packages allowed on the bootclasspath.

(The error only pops up when using "make dist")

Change-Id: Ib17b935ca39de7f6e8e6b68aa81fbebe1d5a30a5
---
 scripts/check_boot_jars/package_allowed_list.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/check_boot_jars/package_allowed_list.txt b/scripts/check_boot_jars/package_allowed_list.txt
index 3b85149b5..30d11d0d5 100644
--- a/scripts/check_boot_jars/package_allowed_list.txt
+++ b/scripts/check_boot_jars/package_allowed_list.txt
@@ -255,3 +255,7 @@ com.qualcomm.wfd
 com.qualcomm.wfd.service
 org.codeaurora.internal
 qcom.fmradio
+
+###################################################
+# IFAA Manager used for Alipay and/or WeChat payment
+org\.ifaa\.android\.manager.*
-- 
2.37.1

